langgraph_supervisor/agent_name.py

```diff
@@ -35,12 +35,13 @@ def add_inline_agent_name(message: BaseMessage) -> BaseMessage:
         return message
 
     formatted_message = message.model_copy()
-    if _is_content_blocks_content(formatted_message.content):
+    if _is_content_blocks_content(message.content):
         text_blocks = [block for block in message.content if block["type"] == "text"]
         non_text_blocks = [block for block in message.content if block["type"] != "text"]
         content = text_blocks[0]["text"] if text_blocks else ""
         formatted_content = f"<name>{message.name}</name><content>{content}</content>"
-        formatted_message.content = [{"type": "text", "text": formatted_content}] + non_text_blocks
+        formatted_message_content = [{"type": "text", "text": formatted_content}] + non_text_blocks
```

Comment:
why change this?

Reply:
This is a weird one. Storing it in an intermediate variable avoids this `mypy` error from assigning the expression directly to `formatted_message.content`:
>Incompatible types in assignment (expression has type "list[dict[str, str]]", variable has type "str | list[str | dict[Any, Any]]")

I think the only alternative would be to explicitly `cast` to the expected type (which I prefer to avoid if possible).